local-setup/tutorial-config.json

```diff
@@ -0,0 +1,52 @@
+{
+  "node": {
+    "bin": "../integritee-node/target/release/integritee-node",
+    "flags": [
+      "--tmp",
+      "--dev",
+      "-lruntime=info",
+      "--ws-port",
+      "9944",
```

Comment:
You're using default values here - why not simply omit them?

Reply:
To be explicit and keep the config valid, even if the default value changes for some reason.